cfgov/preprocessed/js/organisms/ExpandableGroup.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+require( '../modules/polyfill/event-listener' );
+
+var Expandable = require( '../molecules/Expandable' );
+
+/**
+ * ExpandableGroup
+ * @class
+ *
+ * @classdesc Initializes a new Expandable Group organism.
+ *
+ * @param {Object} context
+ *   The DOM element within which to search for the organism.
+ * @returns {Object} An ExpandableGroup instance.
+ */
+function ExpandableGroup( context ) {
```

Comment:
consider changing the name from `context` to `element` as that is actually what it is.